runtime/src/snapshot_package.rs

```diff
@@ -44,28 +36,31 @@ pub struct AccountsPackage {
     pub snapshot_links: TempDir,
     pub storages: SnapshotStorages,
     pub hash: Hash, // temporarily here while we still have to calculate hash before serializing bank
-    pub archive_format: ArchiveFormat,
     pub snapshot_version: SnapshotVersion,
-    pub snapshot_output_dir: PathBuf,
```

Comment:
I was able to remove these two fields, since only `SnapshotPackage` needs them. This will be a small perf boost to not copy a path into here each time, and instead only do it when actually making a SnapshotPackage.